include/RevCPU.h

```diff
@@ -212,17 +212,17 @@ class RevCPU : public SST::Component {
   }
 
 private:
-  unsigned              numCores{};      ///< RevCPU: number of RISC-V cores
-  unsigned              numHarts{};      ///< RevCPU: number of RISC-V cores
-  unsigned              msgPerCycle{};   ///< RevCPU: number of messages to send per cycle
-  unsigned              RDMAPerCycle{};  ///< RevCPU: number of RDMA messages per cycle to inject into PAN network
-  unsigned              testStage{};     ///< RevCPU: controls the PAN Test harness staging
-  unsigned              testIters{};     ///< RevCPU: the number of message iters for each PAN Test
-  RevOpts*              Opts{};          ///< RevCPU: Simulation options object
-  RevMem*               Mem{};           ///< RevCPU: RISC-V main memory object
-  RevLoader*            Loader{};        ///< RevCPU: RISC-V loader
-  std::vector<RevCore*> Procs{};         ///< RevCPU: RISC-V processor objects
-  bool*                 Enabled{};       ///< RevCPU: Completion structure
+  unsigned numCores{};     ///< RevCPU: number of RISC-V cores
+  unsigned numHarts{};     ///< RevCPU: number of RISC-V cores
+  unsigned msgPerCycle{};  ///< RevCPU: number of messages to send per cycle
+  //  unsigned          RDMAPerCycle{};  ///< RevCPU: number of RDMA messages per cycle to inject into PAN network
+  //  unsigned          testStage{};     ///< RevCPU: controls the PAN Test harness staging
+  //  unsigned          testIters{};     ///< RevCPU: the number of message iters for each PAN Test
```

Comment:
These members are currently unused, which caused a warning, and is why I commented them out.